dds/idl/marshal_generator.cpp

```diff
@@ -1753,9 +2183,11 @@ namespace {
         if (accessor) {
           local.erase(local.size() - 2);
         }
-        intro += "  " + getArrayForany(pre.c_str(), name.c_str(), tdname) + '\n';
+        //TODO: Was this redundant call fixed
```

Comment:
TODO

Reply:
@iguessthislldo Did you say you fixed this in your PR?

Reply:
Yes, I turned `intro` into a class with a set of lines and a method to join them together.